terraform/app-monitor-module/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "app_monitor" {
+  type = object({
+    name =string
+    server = string
+    app = string
+    query_level_trigger_id = string
+    automation_queue_action_id = string
+    errormsg = string
```

Comment:
Maybe rename this (and in the json file) as query?

Reply:
set queryblock for query part. Addition to that, use more fields for monitor config